### PR TITLE
gui-desktop : add keyboard shortcut to press OK/Yes button

### PR DESCRIPTION
**Pull request by Ayora29**
gui-desktop : Match : add a configurable keyboard shortcut to press OK/yes button from prompt and pop-ups.
It allows to quickly pass phases and accept stack events.
Default to 'space bar'.  
JButtons already have a default shortcut to space bar by default but for whatever reason it does NOT always works during match.

Should not mess with the rest of the UI because like I said JButtons already have a default shortcut to space bar.

Don't know if I'm allowed to offer this kind of modification.

Don't know If I miss some Panel/pop-ups. In all the match I've done so far, it worked.

### CODE REVIEW

File: forge-gui/res/languages/es-ES.properties [hunk 1]
```diff
@@ -418,6 +418,7 @@ lblSHORTCUT_SHOWTARGETING=Partida: alternar la orientación visual de superposic
 lblSHORTCUT_AUTOYIELD_ALWAYS_YES=Partida: ceder automáticamente en cada habilidad de la pila (Siempre Sí)
 lblSHORTCUT_AUTOYIELD_ALWAYS_NO=Partida: ceder automáticamente en cada habilidad de la pila (Siempre No)
 lblSHORTCUT_MACRO_RECORD=Partida: Grabar una macro de secuencia de acciones
+lblSHORTCUT_PRESS_BUTTON=Partida: Presione el bot�n Aceptar/S�
```
Comment (kevlahnota) [inline forge-gui/res/languages/es-ES.properties:421]: Presione el botón Aceptar/Sí

File: forge-gui/res/languages/it-IT.properties [hunk 2]
```diff
@@ -415,6 +415,7 @@ lblSHORTCUT_SHOWTARGETING=Incontro: attiva / disattiva gli indicatori dei bersag
 lblSHORTCUT_AUTOYIELD_ALWAYS_YES=Incontro: consenso automatico in pila: sempre Sì
 lblSHORTCUT_AUTOYIELD_ALWAYS_NO=Incontro: consenso automatico in pila: sempre No
 lblSHORTCUT_MACRO_RECORD=Incontro: registra una macro (sequenza di azioni)
+lblSHORTCUT_PRESS_BUTTON=Incontro: premere il pulsante OK/S�
```
Comment (kevlahnota) [inline forge-gui/res/languages/it-IT.properties:418]: Premere il pulsante OK/Sì

File: forge-gui/res/languages/de-DE.properties [hunk 3]
```diff
@@ -418,6 +418,7 @@ lblSHORTCUT_AUTOYIELD_ALWAYS_NO=Duel: Auto-Bestätigen von Fähigkeiten auf dem
 lblSHORTCUT_MACRO_RECORD=Duell: Aktion-Abfolge-Makro aufnehmen
 lblSHORTCUT_MACRO_NEXT_ACTION=Duel: führe nächste Aktion im gespeicherten Makro aus
 lblSHORTCUT_CARD_ZOOM=Duell: Zoome ausgewählte Karte
+lblSHORTCUT_PRESS_BUTTON=Duell: Dr�cken Sie die OK/Ja-Taste
```
Comment (kevlahnota) [inline forge-gui/res/languages/de-DE.properties:421]: Drücken Sie die OK/Ja-Taste

File: forge-gui/res/languages/es-ES.properties (hunk 1 above)
Comment (Ayora29) [reply forge-gui/res/languages/es-ES.properties]: Fixed. Thank you

File: forge-gui/res/languages/it-IT.properties (hunk 2 above)
Comment (Ayora29) [reply forge-gui/res/languages/it-IT.properties]: Fixed. Thank you

File: forge-gui/res/languages/de-DE.properties (hunk 3 above)
Comment (Ayora29) [reply forge-gui/res/languages/de-DE.properties]: Fixed. Thank you

**Review by tehdiplomat — APPROVED** [review]

**Comment by Agetian** [comment]
Hmm, the default here seems not to be the best choice - when searching for cards by name, pressing Space confusingly confirms the search instead of letting the search continue. Case in point: easily replicated in dev mode using the "Add Card To Hand" option. Or anything that says "Name a card" - for instance, you can try naming anything with a space in it (e.g. Ugin's Labyrinth).

**Comment by Ayora29** [comment]
Hum I see, I can precent the shortcut to trigger for panel with "lblNameTheCard" (Name the card) title.
Don't know if there is a more elegant way to do this.

**Comment by Agetian** [comment]
That'll probably not catch the all desired cases... In particular, for instance, the dev mode options use other prompts for the window title.

**Comment by Ayora29** [comment]
Indeed. :( Well, I guess I can revert ListChooser.java.
The shortcut will still trigger to pass phases and accept stack events as this iw the main usage for this shortcut.
As Chooser often allows for keyboard input, the shortcut is not a good idea here.

**Comment by Agetian** [comment]
Yeah, I agree. Maybe @kevlahnota or @tehdiplomat can suggest a better solution, but for now, I think reverting ListChooser would be optimal...

**Comment by tehdiplomat** [comment]
I'm just gonna revert the whole thing and we can try again a different way.

**Pull request merged**